### PR TITLE
Update the base image to ubuntu:18.04

### PR DESCRIPTION
**Pull request by superbrothers**
This PR updates the base image to ubuntu:18.04.

### CODE REVIEW

File: ci/pipeline.yml.erb [hunk 1]
```diff
@@ -24,11 +24,11 @@ resource_types:
     tag: latest
 
 resources:
-- name: ubuntu-16.04
+- name: ubuntu-18.04
```
Comment (yuanying) [inline ci/pipeline.yml.erb]: How about making "ubuntu-18.04" be variable?

Comment (superbrothers) [reply ci/pipeline.yml.erb]: Do you mean that `ubuntu-18.04` should be changed to less specific name such as `base-image`?

Comment (yuanying) [reply ci/pipeline.yml.erb]: No,  we are using ERB, so we can use ruby variable.

Comment (superbrothers) [reply ci/pipeline.yml.erb]: Ah, OK. I understand.  I'll do it.

Comment (superbrothers) [reply ci/pipeline.yml.erb]: done. 5f9e1b1

**Pull request merged**